e2e/discoverSheetFlow.spec.js

```diff
@@ -120,18 +120,22 @@ describe('Discover Sheet Flow', () => {
 
   it('Should close search and return to Discover Home on pressing Done', async () => {
     await Helpers.waitAndTap('done-button');
-    await Helpers.checkIfVisible('top-movers-section');
```

Comment:
can we check lists section or something here? otherwise this test is worthless

Reply:
not worthless cause if it fails next tests will fail too!